geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -302,6 +302,7 @@ GPG_KEYS += 1DAFEE6A # Oliver Christen
 GPG_KEYS += C8A0FF72 # Wolfgang Kaltz
 GPG_KEYS += A510717D # Michael Künzli
 GPG_KEYS += A68C2F5F # Arnaud Morvan
+GPG_KEYS += 0545262E5AE04436C83B654D4CEC60DAC4F68132 # Andrea Borghi
```

Comment:
Please use the short key ...

Reply:
@danduk82 Mentioned that there can be conflicts with the short key, notably this has happened to @raphink , which is why he is suggesting to use the long key. What do you think?

Reply:
Then put the long key for everybody...

Reply:
@sbrunner @danduk82 I have put the long key for everybody now